src/Controller.java

```diff
@@ -55,6 +72,53 @@ private void clearFields() {
         theView.setProtein("");
     }
     
+    //to get current date as a string
+    private String currentDate() {
+        SimpleDateFormat sdf = new SimpleDateFormat("EEEE MMMM dd");        
+        String stringDate = sdf.format(new Date());
+        return stringDate;

```

Comment:
no need for this line and defining stringDate , you can just `return sdf.format(new Date());`